src/api/v2/v2AzureResourcesApiWrapper.ts

```diff
@@ -12,10 +12,6 @@ export class V2AzureResourcesApiWrapper implements V2AzureResourcesApi {
         return this.api.apiVersion;
     }
 
-    pickResource<TModel>(options?: ResourcePickOptions | undefined): vscode.ProviderResult<TModel> {
-        return this.callWithTelemetryAndErrorHandling('v2.pickResource', async () => await this.api.pickResource(options));
-    }
-
     revealResource(resourceId: string): Promise<void> {
```

Comment:
Not related to this PR, but I'm inclined toward having this take the item instead of the ID--same as the VSCode API does. Callers can use the `FindTreeItemByIDExperience` to get said item if they only have the ID. There will need to be some translation, as their model item is what they should send, but RG needs to find its own wrapper item and give that to the VSCode `reveal` API.

Reply:
I like that because it keeps our API small (and mimics VS Code API).